src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -2393,6 +2393,9 @@ public static void newTown(Player player, String name, Resident resident, boolea
 			if (TownySettings.hasTownLimit() && dataSource.getTowns().size() >= TownySettings.getTownLimit())
 				throw new TownyException(Translation.of("msg_err_universe_limit"));
 
+			if (TownySettings.getTownAutomaticCapitalisationEnabled())
+				name = name.substring(0, 1).toUpperCase() + name.substring(1);
+			

```

Comment:
It would a bit nicer if this were handled around here:
https://github.com/TownyAdvanced/Towny/blob/d510492b6a3a0b20350e556a8395add00990ef56/src/com/palmergames/bukkit/towny/command/TownCommand.java#L523
so that it can capitalize each word in the town name before we turn the spaces into _'s.

Reply:
Each word will now be capitalised, but kept it at the same spot since it's easier to capitalize with the underscores already there.